src/core/js/views/drawerView.js

```diff
@@ -86,6 +86,19 @@ define([
             this._isCustomViewVisible = true;
             Adapt.trigger('drawer:empty');
             this.showDrawer();
+
+            // Get the class name of the drawer item.
+            var className = view.context.className;
+
+            // Use the class name of the drawer item to retrieve its model from the collection of drawer items.
+            var model = this.collection.find(function(model) {
+                return model.get('className') === className + '-drawer';
+            });
+
+            if (this._hasBackButton) {
```

Comment:
can you please ``` if (this._hasBackButton &&  model.get('title')) ``` or something like this , its always good idea to check current drawer item has title, for example if PLP doesn't have title attribute still its displaying title of resource in drawer of PLP currently